### PR TITLE
fix(hasConfigurableGlobal): handle existing globals with undefined value

### PR DESCRIPTION
**Pull request by Afsoon**
fix: https://github.com/mswjs/interceptors/issues/668

### CODE REVIEW

File: src/utils/hasConfigurableGlobal.test.ts [hunk 1]
```diff
@@ -26,6 +26,15 @@ it('returns false if the global property does not exist', () => {
   expect(hasConfigurableGlobal('_non-existing')).toBe(false)
 })
 
+it('returns false if the global property exists and it\'s value is undefined', () => {
+  Object.defineProperty(global, '_existsAndUndefined', {
+    value: undefined,
+    configurable: true,
+  })
+
+  expect(hasConfigurableGlobal('_existsAndUndefined')).toBe(true)
```
Comment (kettanaito) [inline src/utils/hasConfigurableGlobal.test.ts:35]: This has to be `false` though. Proxying `undefined` will throw an error as it's not an object. 

You are correctly returning `false` in the implementation, but the test still expects `true`. Can you adjust the assertion please?

**Review by kettanaito — APPROVED** [review]
I've fixed the tests, added the check for the root cause at hand (getter that returns undefined). Looks good now!

File: src/utils/hasConfigurableGlobal.test.ts (hunk 1 above)
Comment (Afsoon) [reply src/utils/hasConfigurableGlobal.test.ts]: You are right!

**Comment by kettanaito** [comment]
@Afsoon, thanks for addressing this! Welcome to the contributors 👋

**Comment by kettanaito** [comment]
## Released: v0.36.8 🎉

This has been released in v0.36.8!

- 📄 [**Release notes**](https://github.com/mswjs/interceptors/releases/tag/v0.36.8)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/interceptors/v/0.36.8)

Make sure to always update to the latest version (`npm i @mswjs/interceptors@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.

**Pull request merged**